stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -315,6 +318,23 @@ internal class ChatDomainImpl internal constructor(
         initClean()
     }
 
+    internal fun initRepositoryFacade(user: User) {
```

Comment:
This method isn't invoked anywhere

Reply:
My bad

Reply:
Actually, no need for initRepositoryFacade as ChatClient already does this work.